### PR TITLE
Display ski pistes names

### LINKED ISSUE #1759: Show ski piste name
It would be really helpful to see ski piste names in OM. Right now, we have the piste ways with colors depending on the difficulty which is nice, but having the names displayed would be even better. 

According to the [wiki](https://wiki.openstreetmap.org/wiki/Piste_Maps), pistes are ways with the tags `piste:type` and `piste:difficulty` (and more but these are the basics). If pistes are also used as tracks during other seasons, they are tagged `highway=track` as well. In that case, OM shows the piste name and draws the way dashed. But if a way only has the piste tags, the name will not be shown and the way will be drawn solid. Names are tagged as `name`, but if it is also a track, it may have the `piste:name` tag to differentiate the piste name from the track name.

In the following screenshot, _Boulevard de la griole bas_ is a piste and a track, while the blue line is only a piste.

Screenshot_20211230-212012_Organic_Maps

Here is the link to the piste/track: https://www.openstreetmap.org/way/207012274#map=15/42.6941/1.8020

Here is the location of the piste only (it is not displayed in OSM default renderer, but can be seen in OM and Vespucci): https://www.openstreetmap.org/#map=18/42.69855/1.81307

This would really help users find their way up in the mountains while skiing.

- OS: Android 9
- Organic Maps version: 2021.12.01-4-F-Droid
- Data Version: 211122

### PR DESCRIPTION
**Pull request by pastk**
Added z15- pathtexts.
Adjusted prios and font sizes of aerialways also.

Closes #1759 

image

### CODE REVIEW

File: data/styles/clear/include/Roads_label.mapcss [hunk 1]
```diff
@@ -331,17 +331,40 @@ line|z15-[aerialway=t-bar],
...
+line|z16-17[aerialway=chair_lift],
+line|z16-17[aerialway=drag_lift],
+line|z16-17[aerialway=j-bar],
+line|z16-17[aerialway=t-bar],
+line|z16-17[aerialway=magic_carpet],
+line|z16-17[aerialway=platter],
+line|z16-17[aerialway=rope_tow],
+line|z16-17[piste:type=downhill],
+line|z16-17[piste:type=nordic],
+line|z16-17[piste:type=sled],
```
Comment (vng) [inline data/styles/clear/include/Roads_label.mapcss:351]: nit: z16- is easier to maintain :)

**Review by vng — APPROVED** [review]

File: data/styles/clear/include/Roads_label.mapcss (hunk 1 above)
Comment (pastk) [reply data/styles/clear/include/Roads_label.mapcss:351]: yes, but it slows down styles processing (e.g. the processor will apply z16- rules to z19 even though they'll be overriden later by z18- declaration)

Comment (biodranik) [reply data/styles/clear/include/Roads_label.mapcss:351]: @pastk wow, that's bad! Does it happen in runtime? If yes, can you please create a separate issue for it? If no, and the issue only happens when styles are generated, then it's not a problem.

Comment (pastk) [reply data/styles/clear/include/Roads_label.mapcss:351]: its at the styles generation stage only

Comment (biodranik) [reply data/styles/clear/include/Roads_label.mapcss:351]: Then let's keep it easier for _people_, not for _machines_ :)

**Pull request merged**